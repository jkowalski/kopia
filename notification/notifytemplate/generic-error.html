Subject: Kopia has encountered an error during {{ .EventArgs.Operation }} on {{.Hostname}}

<!doctype html>
<html>
<head>
</head>
<body>

<p><b>Operation:</b> {{ .EventArgs.OperationDetails }}</p>
<p><b>Started:</b> {{ .EventArgs.StartTimestamp }}</p>
<p><b>Finished:</b> {{ .EventArgs.EndTimestamp }} ({{ .EventArgs.Duration }})</p>

<p><b>Message:</b> {{ .EventArgs.ErrorMessage }}</p>

<pre>{{ .EventArgs.ErrorDetails }}</pre>

<p>Generated at {{ .EventTime }} by <a href="https://kopia.io">Kopia {{ .KopiaBuildVersion }}</a>.</p>

</body>
</html>
