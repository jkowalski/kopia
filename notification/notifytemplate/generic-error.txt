Subject: Kopia has encountered an error during {{ .EventArgs.Operation }} on {{.Hostname}}

Operation: {{ .EventArgs.OperationDetails }}
Started:   {{ .EventArgs.StartTimestamp }}
Finished:  {{ .EventArgs.EndTimestamp }} ({{ .EventArgs.Duration }})

{{ .EventArgs.ErrorDetails }}

Generated at {{ .EventTime | formatTime }} by Kopia {{ .KopiaBuildVersion }}.

https://kopia.io/