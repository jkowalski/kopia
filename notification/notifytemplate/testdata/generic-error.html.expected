Subject: Kopia has encountered an error during Some Operation on some-host

<!doctype html>
<html>
<head>
</head>
<body>

<p><b>Operation:</b> Some Operation Details</p>
<p><b>Started:</b> 2020-01-02 03:04:05 +0000 UTC</p>
<p><b>Finished:</b> 2020-01-02 03:04:06 +0000 UTC (1s)</p>

<p><b>Message:</b> error message</p>

<pre>error details</pre>

<p>Generated at 2020-01-02T03:04:05Z by <a href="https://kopia.io">Kopia v0-unofficial</a>.</p>

</body>
</html>
